tests/nnapi/specs/V1_1/transpose_dynamic.mod.py

```diff
@@ -0,0 +1,23 @@
+import dynamic_tensor
```

Comment:
please name this file to `transpose_dynamic_nnfw.mod.py`. I've heard that adding `*_nnfw.mod.py` is the rule to add test code written by us (other `*.mod.py` are from AOSP)

Reply:
ok, will do so, thanks for letting me know.